pallets/creditcoin/src/migrations/v6.rs

```diff
@@ -117,7 +119,7 @@ fn reconstruct_currency(blockchain: &OldBlockchain, kind: &OldTransferKind) -> O
 				.expect("1 is less than the bound (2); qed"),
 		),
 		other => {
-			log::warn!("unexpected transfer kind found in storage: {:?}", other);
+			warn_or_panic!("unexpected transfer kind found in storage: {:?}", other);
```

Comment:
New nightly clippy will complain about this.
```suggestion
			warn_or_panic!("unexpected transfer kind found in storage: {other:?}");
```